src/main/java/com/thirdcc/webapp/service/impl/EventAttendeeServiceImpl.java

```diff
@@ -59,15 +60,15 @@ public EventAttendeeDTO save(EventAttendeeDTO eventAttendeeDTO) {
 
         Event event = eventRepository
             .findOneByIdAndStatusIn(eventAttendeeDTO.getEventId(), eventStatuses)
-            .get();
-
-        if(event != null){
-            EventAttendee eventAttendee = eventAttendeeMapper.toEntity(eventAttendeeDTO);
-            eventAttendee = eventAttendeeRepository.save(eventAttendee);
-            return eventAttendeeMapper.toDto(eventAttendee);
-        } else {
-            return new BadRequestException("This event does not exists or it is not happening"));
+            .orElseThrow(()-> new BadRequestException("This event does not exists or it is not happening"));
+
+        if(event.getEndDate().isBefore(Instant.now())){
```

Comment:
Should it be start or end Date?

Reply:
As discussed in #23 , the validation check will be determine based on the maxAttendees and closingDate.

Reply:
have you implemented this?

Reply:
The #23 will be implement in another pull request.
Validate event with endDate will be used for now.